### ISSUE TITLE
sellersvaiiey

### ISSUE THREAD

**Issue by osama18060124-alt** [osama18060124-alt/osama#4]
<!DOCTYPE html>
<html lang="ar" dir="rtl">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>لوحة تحكم وادي البائعين - Sellers Valley</title>
    <script src="https://cdn.tailwindcss.com"></script>
    <script type="module" src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.4.0/model-viewer.min.js"></script>
    <link href="https://fonts.googleapis.com/css2?family=Tajawal:wght@400;700&display=swap" rel="stylesheet">
    <style>
        body { font-family: 'Tajawal', sans-serif; }
    </style>
</head>
<body class="bg-gray-100">

    <div class="flex h-screen overflow-hidden">
        
        <aside class="w-64 bg-slate-900 text-white flex-shrink-0 hidden md:flex flex-col">
            <div class="p-6 text-2xl font-bold border-b border-slate-800 text-center">
                وادي البائعين
            </div>
            <nav class="flex-1 p-4 space-y-2">
                <a href="#" class="block py-2.5 px-4 rounded transition duration-200 hover:bg-slate-800 bg-slate-800">الرئيسية</a>
                <a href="#" class="block py-2.5 px-4 rounded transition duration-200 hover:bg-slate-800">الطلبات</a>
                <a href="#" class="block py-2.5 px-4 rounded transition duration-200 hover:bg-slate-800 text-blue-400 font-bold">إدارة نماذج 3D</a>
                <a href="#" class="block py-2.5 px-4 rounded transition duration-200 hover:bg-slate-800">المنتجات</a>
                <a href="#" class="block py-2.5 px-4 rounded transition duration-200 hover:bg-slate-800">إحصائيات المبيعات</a>
                <a href="#" class="block py-2.5 px-4 rounded transition duration-200 hover:bg-slate-800">الإعدادات</a>
            </nav>
            <div class="p-4 border-t border-slate-800 text-sm text-center text-gray-400">
                الإصدار 1.0.0
            </div>
        </aside>

        <main class="flex-1 overflow-y-auto">
            <header class="bg-white shadow-sm p-4 flex justify-between items-center px-8">
                <h1 class="text-xl font-bold text-gray-700">إدارة النماذج ثلاثية الأبعاد</h1>
                <div class="flex items-center space-x-reverse space-x-4">
                    <span class="text-sm text-gray-500">مرحباً، المدير</span>
                    <div class="w-10 h-10 bg-blue-600 rounded-full flex items-center justify-center text-white font-bold">م</div>
                </div>
            </header>

            <div class="p-8">
                
                <div class="bg-white rounded-2xl shadow-sm border border-gray-200 overflow-hidden">
                    <div class="p-6 border-b border-gray-100 flex justify-between items-center">
                        <div>
                            <h2 class="text-lg font-bold text-gray-800">معاينة المنتج قبل النشر</h2>
                            <p class="text-sm text-gray-500">قم برفع ملف GLB الخاص بالمعدات للتأكد من التصميم</p>
                        </div>
                        <span class="bg-green-100 text-green-700 px-3 py-1 rounded-full text-xs font-bold">وضع المطور</span>
                    </div>

                    <div class="grid grid-cols-1 lg:grid-cols-3 gap-0">
                        
                        <div class="lg:col-span-2 bg-gray-50 p-4 min-h-[500px] flex items-center justify-center relative">
                            <model-viewer id="main-viewer" 
                                          src="Astronaut.glb" 
                                          alt="معاينة نموذج 3D" 
                                          auto-rotate 
                                          camera-controls 
                                          ar 
                                          shadow-intensity="1" 
                                          style="width: 100%; height: 500px; outline: none;">
                                <button slot="ar-button" class="absolute bottom-4 right-4 bg-white border border-gray-300 px-4 py-2 rounded-lg text-sm shadow-sm">
                                    عرض في مساحتك (AR)
                                </button>
                            </model-viewer>
                        </div>

                        <div class="p-8 bg-white border-r border-gray-100 space-y-6">
                            <div>
                                <label class="block text-sm font-bold text-gray-700 mb-3">رفع ملف جديد</label>
                                <div class="relative border-2 border-dashed border-blue-200 rounded-xl p-6 text-center hover:bg-blue-50 transition cursor-pointer">
                                    <input type="file" id="upload-input" accept=".glb, .gltf" class="absolute inset-0 w-full h-full opacity-0 cursor-pointer">
                                    <svg class="w-8 h-8 text-blue-400 mx-auto mb-2" fill="none" stroke="currentColor" viewBox="0 0 24 24"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M7 16a4 4 0 01-.88-7.903A5 5 0 1115.9 6L16 6a5 5 0 011 9.9M15 13l-3-3m0 0l-3 3m3-3v12"></path></svg>
                                    <p class="text-xs text-gray-500">اسحب الملف هنا أو اضغط للاختيار</p>
                                </div>
                            </div>

                            <div class="space-y-4">
                                <div class="flex justify-between items-center text-sm">
                                    <span class="text-gray-500">اسم الملف:</span>
                                    <span id="file-name-label" class="font-mono text-blue-600 truncate max-w-[150px]">Astronaut.glb</span>
                                </div>
                                <div class="flex justify-between items-center text-sm">
                                    <span class="text-gray-500">الحالة:</span>
                                    <span class="text-green-600">جاهز للمعاينة</span>
                                </div>
                            </div>

                            <div class="pt-6 space-y-3">
                                <button class="w-full bg-blue-600 hover:bg-blue-700 text-white font-bold py-3 rounded-xl transition shadow-lg shadow-blue-100">
                                    نشر في صفحة المتجر
                                </button>
                                <button class="w-full bg-gray-100 hover:bg-gray-200 text-gray-600 font-bold py-3 rounded-xl transition">
                                    حفظ كمسودة
                                </button>
                            </div>
                        </div>
                    </div>
                </div>

            </div>
        </main>
    </div>

    <script>
        const uploadInput = document.getElementById('upload-input');
        const viewer = document.getElementById('main-viewer');
        const fileNameLabel = document.getElementById('file-name-label');

        uploadInput.addEventListener('change', function() {
            const file = this.files[0];
            if (file) {
                const url = URL.createObjectURL(file);
                viewer.src = url;
                fileNameLabel.textContent = file.name;
            }
        });
    </script>
</body>
</html>
